shared/api/server.go

```diff
@@ -16,6 +16,10 @@ type ServerEnvironment struct {
 	ServerVersion          string   `json:"server_version" yaml:"server_version"`
 	Storage                string   `json:"storage" yaml:"storage"`
 	StorageVersion         string   `json:"storage_version" yaml:"storage_version"`
+
+	// API extension: clustering
+	ServerClustered bool   `json:"server_clustered" yaml:"server_clustered"`
+	ServerName      string `json:"server_name" yaml:"server_name"`
```

Comment:
How about we keep those two under /1.0/cluster? (would require a GET /1.0/cluster to retrieve the "enabled" and "server_name" properties)

Reply:
It's used in a few places and would require some change. But sure, if you prefer.

Reply:
Okay, fine to keep where it is, it'll be useful in bug reports as we always request the `lxc info` output.